src/components/header/ReportHeader.jsx

```diff
@@ -7,7 +7,7 @@ import AdReportModal from "@/components/AdReportModal";
 import { useChannelStore } from "@/store/useChannelStore";
 
 const ReportHeader = () => {
-  const channelId = useChannelStore((state) => state.selectedChannelId);
+  const { isChannelChanged, selectedChannelId } = useChannelStore();
```

Comment:
```suggestion
  const isChannelChanged = useChannelStore((state) => state.isChannelChanged);
  const selectedChannelId = useChannelStore((state) => state.selectedChannelId);
```
useChannelStore()를 직접 호출해 구조분해할 경우 store의 모든 상태를 구독하게 되어
다른 값이 바뀌어도 ReportHeader 컴포넌트가 불필요하게 리렌더링될 수 있습니다.

리렌더링을 최소화하기 위해 선택적 구독(selector)을 사용하는 것이 어떨까요?
두 분의 의견이 궁금합니다!
@Eun0713 @devjaeyoon

Reply:
좋은 의견 감사드립니다. `useChannelStore()`를 사용하면 `useChannelStore`의 `interestChannelIds`가 변경될 때 불필요하게 리렌더링되는 상황이 일어날 수 있겠네요. 해당 사항 반영하겠습니다!

Reply:
저도 `selector` 방식이 더 좋다고 생각해요!
전체 store를 구조분해하면 다른 값 변경 시에도 리렌더링될 수 있어서 필요한 값만 구독하는 게 더 효율적인 것 같습니다😊